backend/zeno_backend/server.py

```diff
@@ -404,18 +386,29 @@ def get_project_state(
         return select.project_state(project_uuid, user, project)
 
     @api_app.get(
-        "/report/{owner}/{report}", response_model=ReportResponse, tags=["zeno"]
+        "/report-name/{owner_name}/{report_name}",
+        response_model=ReportResponse,
+        tags=["zeno"],
     )
-    def get_report(owner_name: str, report_name: str, request: Request):
-        rep = select.report(owner_name, report_name, util.get_user_from_token(request))
+    def get_report_by_name(owner_name: str, report_name: str, request: Request):
+        report_id = select.report_id(owner_name, parse.unquote(report_name))
```

Comment:
OOC: do we need this parse? Asking since we did not have it before. Also, would it make sense to add for owner_name as well?

Reply:
I think we do... can update the other ones.